config/persistence.php

```diff
@@ -18,4 +22,13 @@
                 tagged_iterator('.foundry.persistence.schema_resetter'),
             ])
     ;
+
+    if (PHP_VERSION_ID >= 80400) {
+        $container->services()->set('.foundry.persistence.objects_tracker', PersistedObjectsTracker::class)
+            ->tag('kernel.reset', ['method' => 'refresh'])
+            ->tag('kernel.event_listener', ['event' => TerminateEvent::class, 'method' => 'refresh'])
+            ->tag('kernel.event_listener', ['event' => ConsoleTerminateEvent::class, 'method' => 'refresh'])
```

Comment:
`ConsoleTerminateEvent` but you mentionned that it is with zenstruck/console-test`, so let's keep it this way